### ISSUE TITLE
Scribe tool wish list

### ISSUE THREAD

**Issue by kimdhamilton** [w3c-ccg/meetings#23]
Extensions of @rhiaro's scribe_tool improvements:

* automate cleanup ui -> git - I think scribe-tool can already talk to
the git API, just needs hooking up to the the frontend and some api
token magic..
* have ccg-bot automatically offer up the cleanup URL in IRC once the
call ends for the designated logs-cleaner to click on (in SocialWG this
used to also be the scribe's job - and I made a similar web tool for
that too - so it was a different person every week, though in practice I
ended up doing it a lot of the time anyway).
* make the scribe-tool UI actually look nice.

**Comment by rhiaro** [comment]
More ideas:

* warnings in the UI if agenda and scribe are missing
* highlight in the UI where `s/../...` is present
* best guess at correcting the `s/../..` automatically (with a highlight for checking)
* warning in the UI for unknown aliases (warnings are currently in the console)

**Comment by rhiaro** [comment]
Would adding the raw audio download link in the minutes cleanup UI be helpful?

**Comment by kimdhamilton** [comment]
> Would adding the raw audio download link in the minutes cleanup UI be helpful?

yes, this is a great idea. Do you happen to know -- in jitsi are we storing as ogg by default? Previously we'd get a .wav and convert to .ogg

**Comment by rhiaro** [comment]
Yes, I think the jitsi audio is stored as ogg

**Comment by rhiaro** [comment]
* [bug] scribe-tool is overeager at guessing peoples' names, and attributing/presenting people who aren't actually in the meeting when someone with the same first name but no entry in people.json is present
* [bug] formatting is wrong for first line of scribed speech attributed to someone with no people.json entry

**Comment by rhiaro** [comment]
[bug] scribe-tool is not getting people who present+ but have no entry in people.json. In this case, it should add the nick to the present list

**Comment by rhiaro** [comment]
[bug] *sometimes* lines written in IRC appears as scribed lines attributed to the person in IRC. Not consistent though! Infuriating.
